### PR TITLE
ZEPPELIN-1324: Make paragraph code selectable when running

### PR DESCRIPTION
**Pull request by karup1990**
### What is this PR for?
Make paragraph code selectable when running(*PENDING* or *RUNNING* state)

### What type of PR is it?
 Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1324

### How should this be tested?
Try to select, copy, paste the paragraph code, when paragraph is running

### Questions:
* Does the licenses files need update? NA
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by karup1990** [comment]
Failure seems unrelated.
@Leemoonsoo @corneadoug This PR is ready for reviewing.

**Comment by corneadoug** [comment]
Using watchers isn't recommended, you can do it in a more simple way.
All that is needed is to do:
* `$scope.editor.setReadOnly($scope.isRunning());` [here](https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js#L2519) and [there](https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js#L93)

* Replace the `disable` from ng-class by a new class that set `opacity: 0.6`

**Comment by karup1990** [comment]
@corneadoug Addressed feedbacks.
Ready for review.

**Comment by corneadoug** [comment]
@karup1990 Tested, works great
LGTM after fixing my inline comment

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
     $scope.chart = {};
     $scope.colWidthOption = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
     $scope.paragraphFocused = false;
+    $scope.editor.setReadOnly($scope.isRunning());
```
Comment (corneadoug) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: Sorry, my bad for pointing out that place, editor is not loaded yet at that point, so we have an error in the console. The right place should be around Line 542

**Comment by corneadoug** [comment]
CI failure is not relevant.
Merging if there is no more discussions

**Pull request closed without merging**